docs/rfcs/blocks-in-widget-areas.md

```diff
@@ -0,0 +1,520 @@
+# Blocks in Widget Areas RFC
+
+This RFC outlines the technical approach that we will take to upgrade the widget-editing areas in `wp-admin/widgets.php` and the Customizer to support blocks.
+
+Adding blocks to widget-editing areas was first announced at WCUS 2018, is a [Gutenberg Phase 2][phase-2] project, and appears in Matt's list of [9 Projects for 2019][2019-projects].
+
+![widgets](https://user-images.githubusercontent.com/1202812/51921660-d4ef0e00-23b5-11e9-86c9-5cb25c224d0b.gif)
```

Comment:
Screenshot of mock-up of customiser?

Reply:
I didn't want to go overboard with graphics. It's just a little bit of context for the more visually minded among us. There's a screenshot of the Customizer in https://github.com/WordPress/gutenberg/issues/13205 if you're curious, though.